server/ruby/inventory.rb

```diff
@@ -0,0 +1,52 @@
+require 'stripe'
+require 'dotenv'
+
+Dotenv.load(File.dirname(__FILE__) + '/../../.env')
+
+Stripe.api_key = ENV['STRIPE_SECRET_KEY']
+Stripe.api_version = '2019-02-11'
+
+# For product retrieval and listing set API version to 2018-02-28 so that skus are returned.
+@@product_api_stripe_version = '2018-02-28'
+
+class Inventory
+  def self.calculate_payment_amount(items)
+    Stripe.api_version = @@product_api_stripe_version
```

Comment:
We should use per request configuration instead: https://github.com/stripe/stripe-ruby#per-request-configuration